mslib/msui/mscolab_admin_window.py

```diff
@@ -238,12 +242,13 @@ def delete_selected_users(self):
         }
         url = url_join(self.mscolab_server_url, "delete_bulk_permissions")
         res = requests.post(url, data=data)
-        res = res.json()
-        if res["success"]:
-            self.load_users_without_permission()
-            self.load_users_with_permission()
-        else:
-            self.show_error_popup(res["message"])
+        if res.text != "False":
```

Comment:
@ReimarBauer for this request we return JSON response. I believe this false check is only when user's token is expired? If so in else should we not call the logout state?

Reply:
> @ReimarBauer for this request we return JSON response. I believe this false check is only when user's token is expired? If so in else should we not call the logout state? Moreover instead of checking for False maybe we can check with status code instead?

@Tanish0019 
Yes, on expired.

I think the logout simplifies everything, all other widgets gets closed and can't create various popup messages, only one is needed to inform the user. 

I change to status code.

Reply:
Wait, the problem is that we don't return json only "False" on text, so there is no way to create json from that and look on the status code.

Reply:
Currently I use only a popup message. further improvements should be done with  https://github.com/Open-MSS/MSS/issues/975